osbs/utils/__init__.py

```diff
@@ -726,3 +726,39 @@ def copy(self):
             namespace=self.namespace,
             repo=self.repo,
             tag=self.tag)
+
+
+class UserWarningsStore(object):
+    def __init__(self):
+        # asctime (platform:arch)? - name - levelname - message
+        self.regex = r'^.+ - USER_WARNING - (\{.*\})$'
```

Comment:
Loglevel name should be constant in `osbs.constants` and that constant should be used here instead of hardcoding the same thing on several places
https://github.com/containerbuildsystem/osbs-client/blob/master/osbs/__init__.py#L19